stable/etcd-operator/templates/NOTES.txt

```diff
@@ -1,19 +1,28 @@
-{{- if .Values.cluster.enabled -}}
+{{- $clusterEnabled := (and (not .Release.IsInstall) .Values.cluster.enabled) -}}
+{{- if and .Release.IsInstall .Values.cluster.enabled -}}
+Not enabling cluster, the ThirdPartResource must be installed before you can create a Cluster. Continuing rest of normal deployment.
+
+{{ end -}}
+
+{{- if $clusterEnabled -}}
 1. Watch etcd cluster start
   kubectl get pods -l etcd_cluster={{ .Values.cluster.name }} --namespace {{ .Release.Namespace }} -w
 2. Confirm etcd cluster is healthy
-  $ kubectl run --rm -i --tty etcd-test --image quay.io/coreos/etcd --restart=Never -- /bin/sh
-  / # ETCDCTL_API=3 etcdctl --endpoints http://{{ .Values.cluster.name }}-0000:2379 put foo bar
-  / # ETCDCTL_API=3 etcdctl --endpoints http://{{ .Values.cluster.name }}-0000:2379 member list
+  $ kubectl run --rm -i --tty --env="ETCDCTL_API=3" --env="ETCDCTL_ENDPOINTS=http://{{ .Values.cluster.name }}-0000:2379" etcd-test --image quay.io/coreos/etcd --restart=Never -- /bin/sh -c 'watch -n1 "etcdctl  member list"'
+
+3. Interact with the cluster!
+  $ kubectl run --rm -i --tty --env ETCDCTL_API=3 etcd-test --image quay.io/coreos/etcd --restart=Never -- /bin/sh
+  / # etcdctl --endpoints http://{{ .Values.cluster.name }}-0000:2379 put foo bar
+  / # etcdctl --endpoints http://{{ .Values.cluster.name }}-0000:2379 get
```

Comment:
(Suggestion) it would be preferable to append namespace to endpoints, as the test pod (etcd-test) could be deployed in a different namespace than the etcd-cluster's namespace.
Also is it possible to use just the cluster.name instead of appending -0000, as it looks ugly. something like
`etcdctl --endpoints http://{{ .Values.cluster.name }}{{ .Release.Namespace }}:2379 put foo bar`